src/content/en/2020/accessibility.md

```diff
@@ -122,7 +122,7 @@ The top 5 file extensions explicitly included in the `alt` text value (for sites
     {{ figure_link(
       caption="Percent of pages with non-empty alt attributes.",
       sheets_gid="TODO",
-      sql_file="TODO.sql"
+      <!-- sql_file="TODO.sql" -->
```

Comment:
Can't do this.

Can we do this instead?:

```
{# TODO add sheets and sql #}

{{ figure_link(
      caption="Percent of pages with non-empty alt attributes.",	      caption="Percent of pages with non-empty alt attributes.",
) }}
  </figcaption>
</figure>	</figure>
```

Note need a blank line after the Jina2 `{# TODO... #}`